101-iothub-device-provisioning/azuredeploy.json

```diff
@@ -35,17 +35,22 @@
                 "description": "The number of IoT Hub units."
             }
         },
-        "requireTLS": {
+        "minTlsVersion": {
             "type": "bool",
-            "defaultValue": false,
+            "defaultValue": "1.2",
+            "allowedValues": [
+                "1.0",
+                "1.1",
```

Comment:
1.0 and 1.1 won't be valid options in a DPS quickstart template

Reply:
ok, if this template always deploys DPS and only 1.2 is allowed for DPS, then we should just set TLS to 1.2 all the time - no conditionals based on location (or anything else)...  what am I missing?